ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -145,6 +145,12 @@ argparse.add_argument(
     help='AWS secret access key (Optional)',
     metavar='AWS_SECRET_KEY'
 )
+argparse.add_argument(
+    '-o', '--session-token',
```

Comment:
Either use "t" as the short name or remove the short option, "o" would make me think of "output"

Reply:
-t is already used by type